geode-dunit/src/main/java/org/apache/geode/management/internal/cli/commands/QueryCommandDUnitTestBase.java

```diff
@@ -200,6 +212,14 @@ private static void prepareDataForRegion(String regionPath) {
     }
   }
 
+  private static void prepareDataForRegionWithSpecialCharacters(String regionPath) {
+    InternalCache cache = ClusterStartupRule.getCache();
+    Region dataRegion = cache.getRegion(regionPath);
```

Comment:
Please try to use non-raw types in places like this. There's an IntelliJ inspection you can enable that'll highlight lines like this:
```
Region<Integer, String> dataRegion = cache.getRegion(regionPath);

dataRegion.put(1, "value$");
dataRegion.put(2, "value%");
```

Reply:
Don't worry about the other lines in this old class that use raw types.